modules/cluster-logging-dashboards-es.adoc

```diff
@@ -0,0 +1,196 @@
+
+|Cluster unassigned shards
+|The number of unassigned shards. Elasticsearch shards might be unassigned for reasons such as a new index being added or the failure of a node.
+
+|===
+
+Elasticsearch node metrics::
+
+Each Elasticsearch node has a finite amount of resources that can be used to process tasks. When all the resources are being used and Elasticsearch attempts to perform a new task, Elasticsearch put the tasks into a queue until some resources become available. 
+
+The *Logging/Elasticsearch Nodes* dashboard contains the following charts about resource usage for a selected node and the number of tasks waiting in the Elasticsearch queue.
+
+.Elasticsearch node metric charts
+[options="header"]
+|===
+|Metric|Description
+
+|ThreadPool tasks
+|The number of waiting tasks in individual queues, shown by task type. A long–term accumulation of tasks in any queue could indicate node resource shortages or some other problem.
+
+|CPU usage 
+|The amount of CPU being used by the selected Elasticsearch node as a percentage of the total CPU allocated to the host container.
+
+|Memory usage 
+|The amount of memory being used by the selected Elasticsearch node.
+
+|Disk usage 
+|The total disk space being used for index data and metadata on the selected Elasticsearch node. 
+
+|Documents indexing rate 
+|The rate that documents are indexed on the selected Elasticsearch node.
+
+|Indexing latency 
+|The time taken to index the documents on the selected Elasticsearch node. Indexing latency can be affected by many factors, such as JVM Heap memory and overall load. A growing trend indicates a resource capacity shortage in the instance.
+
+|Search rate 
+|The number of search requests executed on the selected Elasticsearch node.
+
+|Search latency  
+|The time taken to complete search requests on the selected Elasticsearch node. Search latency can be affected by many factors. A growing trend indicates a resource capacity shortage in the instance.
+
+|Documents count (with replicas) 
+|The number of Elasticsearch documents stored on the selected Elasticsearch node, including documents stored in both the primary shards and replica shards that are allocated on the node.
+
+|Documents deleting rate 
+|The number of Elasticsearch documents being deleted from any of the index shards allocated to the selected Elasticsearch node.
+
+|Documents merging rate 
+|The number of Elasticsearch documents being merged in any of index shards allocated to the selected Elasticsearch node.
+
+|===
+
+Elasticsearch node fielddata::
+
+_Fielddata_ is a query-time in-memory data structure for text fields that is built by Elasticsearch at the index-segment level and kept in the JVM Heap. Because fielddata building is an expensive operation, Elasticsearch caches the fielddata structures. Elasticsearch can evict a fielddata cache when the underlying index segment is deleted or merged, or if there is not enough JVM HEAP memory for all the fielddata caches. 
```

Comment:
I don't think the first sentence is going to translate well. Can you use simpler language?
```suggestion
_Fielddata_ is a query-time in-memory data structure for text fields that is built by Elasticsearch at the index-segment level and kept in the JVM Heap. Because fielddata building is an expensive operation, Elasticsearch caches the fielddata structures. Elasticsearch can evict a fielddata cache when the underlying index segment is deleted or merged or if there is not enough JVM HEAP memory for all the fielddata caches. 
```